DevIncubator_Autopark/AllCollections/MyQueue.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace DevIncubator_Autopark.AllCollections
+{
+    public class MyQueue<T> : IEnumerable<T>
+    {
+        private const int DefaultCapacity = 10;
+        private const int ResizeValue = 2;
```

Comment:
this variable use only in one method, why you create field?

Reply:
Fixed